docs/source/content/overview/overfitting.rst

```diff
@@ -9,26 +9,100 @@ Overfitting and Underfitting
+(Created using Wolfram Alpha)
+
+----------------------------
+Motivation
+----------------------------
+
+Finding a good fit is one of the central problems in machine learning.
+Gaining a good grasp of how to avoid fitting problems before even worrying
+about specific methods can keep models on track.
+The mindset of hunting for a good fit, rather than throwing more learning
+time at a model is very important to have.
 
+----------------------------
+Code
+----------------------------
+
+The example code for overfitting shows some basic examples based in polynomial
+interpolation, trying to find the equation of a graph.
+The overfitting.py_ file, you can see that there is a true function being
+modeled, as well as some estimates that are shown to not be accurate.
+
+.. _overfitting.py: https://github.com/machinelearningmindset/machine-learning-course/blob/master/code/overview/overfitting/overfitting.py
+
+The estimates are representations of overfitting and underfitting.
+For overfitting, a higher degree polynomial is used (x cubed instead of squared).
+While the data is relatively close for the chosen points, there are some artifacts outside of them.
+The example of underfitting, however, does not even achieve accuracy at many of the points.
+Underfitting is similar to having a linear model when trying to model a quadratic function.
+The model does well on the point(s) it trained on, in this case the point used
+for the linear estimate, but poorly otherwise.
 
-The issue with this estimate is clear to the human eye, the model should be nonlinear, and is instead just a simple line.  In machine learning, this could be a result of underfitting, the model has not had enough exposure to training data to adapt to it, and is currently in a simple state.
 
 ----------------------------
-How to avoid overfitting
+Conclusion
 ----------------------------
-A key idea in avoiding overfitting issues in machine learning is to maintain a **validation set**.  This set is used for training purposes, but most importantly the model **does not learn from it**.  So, the model first learns from the **training set**, then checks its knowledge on a completely different **validation set**.  When it performs well enough on the **validation set**, we can be more confident that it is not overfit to the training data than if we just looked at training results.
+
+Check out the cross-validation and regularization sections for information on
+how to avoid overfitting in machine learning models.
+Ideally, a good fit looks something like this:
+
+.. figure:: _img/GoodFit.PNG
+   :scale: 50 %
+   :alt: Underfit
+(Created using Wolfram Alpha)
+
+
+When using machine learning in any capacity, issues such as overfitting
+frequently come up, and having a grasp of the concept is very important.
+The modules in this section are among the most important in the whole repository,
+ since regardless of the implementation, machine learning always includes these fundamentals.
```

Comment:
This appears as a quote since it has a space at the front

Reply:
9185ec3